vignettes/algorithm_data.Rmd

```diff
@@ -0,0 +1,151 @@
+---
+title: "Description of data objects related to the algorithm"
+output: rmarkdown::html_vignette
+bibliography: references.bib
+csl: vancouver.csl
+vignette: >
+  %\VignetteIndexEntry{Design}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+library(dplyr)
+```
+
+## Contents
+
+This document describes the structure of the data components processed
```

Comment:
I have used the word "data sources". Do we prefer "data components"? Or "data objects" as in the title? I think it makes sense to align this, so the same words are used for the same things throughout the documentation :)